### DISCUSSION TITLE
Add multiple repos to shortcuts scope

### DISCUSSION THREAD

**Discussion by Bad3r** [community/community#10110] [Mobile] [answered]
Hi👋,
This is a feature request to allow selecting multiple repos as the scope for a shortcut. 

Currently the scope is either limited to a single repository or all repos. Pleas allow adding a custom amount of repos to scope. 
![image](https://user-images.githubusercontent.com/25513724/149441255-0ff9cd00-694e-4248-9803-0fead59f30c9.png)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the suggestion @Bad3r, I agree that would be useful — I've created a tracking issue for our team so that we can prioritize improvements to shortcuts in the future.
